pom_confluent.xml

```diff
@@ -215,6 +215,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.0</version>
                 <configuration>
+                    <argLine>--add-opens=java.base/java.nio=ALL-UNNAMED</argLine>                
```

Comment:
@acristu related to the change to the compiler target language level: Java 17?

Reply:
yes, we need this for unit tests, in production we have this already set on the connect process